src/LayerAnalyser.php

```diff
@@ -41,20 +41,20 @@ public function analyse(Configuration $configuration, string $layer): array
         $classLikeNames = [];
 
         foreach ($astMap->getAstClassReferences() as $classReference) {
-            $classLikeName = $classReference->getClassLikeName();
+            $classLikeName = $classReference->getTokenLikeName();
             if ($this->isInLayer($layer, $classLikeName, $classLikeLayerResolver)) {
```

Comment:
Why not check instance of here, but below?

Reply:
Because `getAstClassReferences()` will always return `ClassLikeName`, so you do not need to check for it. The other function can return `TokenLikeName`, co you need to check for it being `ClassLikeName` right now.